python_modules/libraries/dagster-dbt/dagster_dbt/core/resources_v2.py

```diff
@@ -543,6 +613,7 @@ def run(
         raise_on_error: bool,
         context: Optional[OpExecutionContext],
         adapter: Optional[BaseAdapter],
+        postprocessing_tasks: Optional[List[DbtPostprocessingAsyncTask]],
```

Comment:
Just so we have maximum flexibility, we shouldn't expose this API to the user yet. Once we have confidence in this scheme working for out-of-the-box metadata (e.g. row count), then we can consider exposing it to the user so they also have more flexibility.